templates/main/login.html

```diff
@@ -7,6 +7,29 @@
         <script src="js/login.js" defer></script>
     </head>
     <body>
+    	<div id='container'>
         <h1>Login page</h1>
+        <form action="" method="POST">
+			  
+			  	
+			    <label for="uname"><b>Username</b></label>
+			    <input type="text" placeholder="Enter Username" name="uname" required>
+			    <br>
+			    <label for="psw"><b>Password</b></label>
+			    <input type="password" placeholder="Enter Password" name="psw" required>
+			    <br>
+			    <button type="submit" name='submitbutton'>Login</button>
+			    <br>
+			    <br>
+			    <h10>Don't have an account? <a href="signup.php">Sign up</a></h10>
```

Comment:
`h10` is not a heading defined by the HTML Doctype (there are only `h1`-`h6`). However it is somehow valid in HTML5 and as a result browsers render it as plain text. There is no styling applied to it as well which prompts me to think that it can be replaced with a different tag, like `span`, `em`, `strong`, `p`, etc.